installing/installing_bare_metal/preparing-to-install-on-bare-metal.adoc

```diff
@@ -0,0 +1,27 @@
+[id="preparing-to-install-on-bare-metal"]
+= Preparing for bare metal cluster installation
+include::modules/common-attributes.adoc[]
+:context: preparing-to-install-on-bare-metal
+
+toc::[]
+
+[id="preparing_preparing-to-install-on-bare-metal"]
+== Prerequisites
+
+* You reviewed details about the xref:../../architecture/architecture-installation.adoc#architecture-installation[{product-title} installation and update] processes.
+
+* You have read the documentation on xref:../../installing/installing-preparing.adoc#installing-preparing[selecting a cluster installation method and preparing it for users].
+
```

Comment:
Would you mind removing the extra line break?

Reply:
Glad to do so!